identity_iota/src/did/did.rs

```diff
@@ -131,8 +131,19 @@ impl IotaDID {
 
     pub fn normalize(&mut self) {
         match &*self.id_segments {
-            [_] => self.id_segments.insert(0, Self::NETWORK.into()),
-            [_, _] | [_, _, _] => {}
+            [_] => {}
+            [network, _] => {
```

Comment:
you can use the same style of pattern as before -> `[network, _] | [network, _, _] => {`

Reply:
Thanks, forgot the shard DID the first time that's why I had them separated